decentraland/helpers/storage.js

```diff
@@ -0,0 +1,59 @@
+import { SecureStore } from "expo";
+import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
+
+// Storage keys
+const EPHEMERAL_ACCOUNT_PRIV_KEY = "EPHEMERAL_ACCOUNT_PRIV_KEY";
+const MY_ASSETS_LIST = "MY_ASSETS_LIST";
+
+export const storeEphemeralAccount = async account => {
+  const { privateKey } = account;
+  await _storeData(EPHEMERAL_ACCOUNT_PRIV_KEY, privateKey);
+};
+
+export const retrieveEphemeralAccount = async () => {
+  let account = null;
+
+  const privateKey = await _retrieveData(EPHEMERAL_ACCOUNT_PRIV_KEY);
+  if (privateKey != null) account = createFromPrivateKey(privateKey);
+
+  return account;
+};
+
+export const storeMyAssets = async myAssets => {
+  const strMyAssets = JSON.stringify(myAssets);
```

Comment:
If we wanted to we could have gotten pretty fine-grained with splitting out the object into keys and values and persisting them separately. Thoughts on pros/cons?

Reply:
This won't gate merging

Reply:
Not strong opinion, I think that we should keep it in mind and do that when it'll help us with some feature that demands to deal with the assets in that manner.

Reply:
Yep, sounds good. Resolving this!